### PR TITLE
Rename Cloud to Connectivity Services & split Apps in Apps and Marketplace

### PR DESCRIPTION
**Pull request by geekgonecrazy**

### CODE REVIEW

**Comment by engelgabriel** [comment]
I need to do some more tests with this.

**Comment by rodrigok** [comment]
Why this was closed?

**Comment by geekgonecrazy** [comment]
In its current state I think it will introduce more confusion then it solves.  I don't think we've come up with a way to improve this in the time left for 1.0.

I'll reopen for now though I guess :)

**Comment by engelgabriel** [comment]
I think the only way to address the problem of hiding things, is to have the 3 main 

Apps
Marketplace
Connectivity Services (current Cloud)

File: app/apps/client/admin/apps.html [hunk 1]
```diff
@@ -1,21 +1,18 @@
 <template name="apps">
 	<section class="rc-apps-marketplace">
-		{{#header sectionName="Apps" hideHelp=true fixedHeight=true fullpage=true}}
-			{{#if appsDevelopmentMode}}
-				<button class="rc-button rc-button--small rc-button--primary rc-button--outline" data-button="install">
-					{{> icon icon="upload" block="rc-icon--default-size"}} {{_ "Upload app"}}
+		{{#header sectionName="Marketplace" hideHelp=true fixedHeight=true fullpage=true}}
+			<div class="rc-header__section-button">
+				<button class="rc-button rc-button--small rc-button--primary" data-button="install_app">
+					{{> icon icon="cloud-plus" block="rc-icon--default-size"}} {{_ "Install_app"}}
```
Comment (geekgonecrazy) [inline app/apps/client/admin/apps.html]:
```suggestion
					{{> icon icon="cloud-plus" block="rc-icon--default-size"}} {{_ "Marketplace_view_marketplace"}}
```

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 2]
```diff
@@ -1564,6 +1565,7 @@
   "initials_avatar": "Initials Avatar",
   "inline_code": "inline code",
   "Install": "Install",
+  "Install_app": "Install App",
```
Comment (geekgonecrazy) [inline packages/rocketchat-i18n/i18n/en.i18n.json]:
```suggestion
  "Marketplace_view_marketplace": "View Marketplace",
```

**Pull request merged**